mlflow/R/mlflow/tests/testthat/helpers.R

```diff
@@ -11,3 +11,26 @@ deregister_local_servers <- function() {
   purrr::walk(as.list(mlflow:::.globals$url_mapping), ~ .x$handle$kill())
   rlang::env_unbind(mlflow:::.globals, "url_mapping")
 }
+
+as_uri <- function(file_path) {
+  if (.Platform$OS.type == "windows") {
+    # normalize file path such as c:/directory into file://c:/directory on
+    # Windows so that MLflow CLI does not confuse 'c' as the scheme of the URI
+    # Also, replacing back slashes with forward slashes
+    gsub("\\\\", "/", paste0("file://", normalizePath(file_path)))
+  } else {
+    # UNIX file paths such as ., dir1, dir1/dir2, / or, /dir1/dir2 etc do not
+    # have this problem so we don't need to do anything
+    file_path
+  }
+}
+
+skip_on_windows <- function(reason = NULL) {
```

Comment:
I think we can fallback to `testthat::skip_on_os()`.

Reply:
Yes, but `testthat::skip_on_os()` currently doesn't pass along a reason string.